ops/jujuversion.py

```diff
@@ -127,6 +127,14 @@ def has_controller_storage(self) -> bool:
         """Report whether this Juju version supports controller-side storage."""
         return (self.major, self.minor, self.patch) >= (2, 8, 0)
 
+    def has_pebble_identities(self) -> bool:
```

Comment:
Thanks for adding these. We've been inconsistent in naming these over time (and whether they're functions or properties). However, I think we should follow the more recent convention: move these to the button, call them `supports_x` instead of `has_x`, and make them properties instead of methods.